CONTRIBUTORS.md

```diff
@@ -27,6 +27,8 @@ involved in the project:
 * John Smolin
 * Erick Winston
 * Chris Wood
+* Takashi Imamichi
```

Comment:
can you insert these in alphabetical order?

Reply:
Yes, I will do it.

Reply:
done